src/main/webapp/WEB-INF/view/activity.jsp

```diff
@@ -7,8 +7,9 @@
 <body>
   <%@ include file = "/navigations.jsp" %>
 
-  <div id="activityfeed">
-    <h2 style="color:blue"> WELCOME TO THE <strong> ACTIVITY FEED PAGE </strong> (in progress) </h2>
+  <div id="container">
+    <h2 style="color:blue" "text-aligned:left" "margin-bottom:25px"> ACTIVITY </h2>
```

Comment:
I think this should be `style="color:blue; text-aligned:left; margin-bottom:25px"` ?

Reply:
Even if it is working?

Reply:
It's actually only picking up the information in the first set of quotes.  If you switch the order of the attributes around so that `color:blue` is last, you'll see that it's no longer making the text blue.

Reply:
oh ok; thanks. It looked like it was working. CSS should at least tell me. Thanks.